content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,364 @@
+cd $HOME/kubernetes && git checkout v1.14.0
+make
+echo alias kubectl='$HOME/kubernetes/cluster/kubectl.sh' >> /etc/profile
+hack/install-etcd.sh
+ENABLE_DAEMON=true ALLOW_PRIVILEGED=true FEATURE_GATES=VolumeSnapshotDataSource=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh -O
+```
+
+### [OpenSDS](https://github.com/sodafoundation/opensds) local cluster
+
+For testing purposes you can deploy OpenSDS refering
+to [OpenSDS Cluster Installation through Ansible](https://github.com/sodafoundation/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible)
+.
+
+## Testing steps
+
+##### Following are some additional steps we need to perform for CEPH-RBD CSI Plugin.
+
+* Create a block image/volume inside ceph cluster.
+
+```
+rbd create osdsrbd/vol1 --size 100 --order 12
+```
+
+* Mount a block image.
+
+```
+rbd map osdsrbd/vol1 --id admin
+```
+
+* Adjust the tunables on the ceph cluster.
+
+```
+ceph osd crush tunables hammer
+
+grep -q "^rbd default features" /etc/ceph/ceph.conf || sed -i '/\[global\]/arbd default features = 1' /etc/ceph/ceph.conf
+```
+
+* Change the workplace
+
+  ```
+  cd /opt/opensds-sushi-linux-amd64
+  ```
+* Opensds supports csi plugin for both block and file storage types
+
+* For csi block plugin:
+
+    * Update profile id of StorageClass in csi/examples/kubernetes/block/nginx.yaml according to actual profile you
+      created, it looks like this:
+      ```
+      apiVersion: storage.k8s.io/v1
+      kind: StorageClass
+      metadata:
+        name: csi-sc-opensdsplugin-block
+      provisioner: csi-opensdsplugin-block
+      parameters:
+        attachMode: rw
+        profile: d10ec339-3357-43ff-8626-4ccdb854af3d
+      ```
+    * Create example nginx application
+
+      ```
+      kubectl create -f csi/examples/kubernetes/block/nginx.yaml
+      ```
+
+* For csi file plugin:
+
+    * Update profile id of StorageClass in csi/examples/kubernetes/file/nginx.yaml according to actual profile you
+      created, it looks like this:
+      ```
+      apiVersion: storage.k8s.io/v1
+      kind: StorageClass
+      metadata:
+        name: csi-sc-opensdsplugin-file
+      provisioner: csi-opensdsplugin-file
+      parameters:
+        attachMode: rw
+        profile: d10ec339-3357-43ff-8626-4ccdb854af3d
+      ```
+    * Create example nginx application
+
+      ```
+      kubectl create -f csi/examples/kubernetes/file/nginx.yaml
+      ```
+  This example will mount a opensds volume into `/var/lib/www/html`. You can use the following command to inspect into
+  nginx container to verify it.
+
+  ```
+  kubectl exec -it nginx /bin/bash
+  ```
+
+## Clean up steps
+
+Clean up example nginx application and opensds CSI pods by the following commands:
+
+* For csi block plugin:
+
+  ```
+  kubectl delete -f csi/examples/kubernetes/block/nginx.yaml
+  kubectl delete -f csi/deploy/kubernetes/block
+  ```
+* For csi file plugin:
```

Comment:
csi -> CSI.
Please check this across the document.

Reply:
sure @anvithks